### PR TITLE
Changed how compose is parsed from Jenkins API object

### LINKED ISSUE #153: Composes are no longer being parsed correctly
Due to a change in the Jenkins API object being returned to Jeeves, the compose IDs being parsed from each job are no longer returning correctly

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Fixes #153 

Fix worked for Enterprise DFG jobs

### CODE REVIEW

File: functions.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def get_jenkins_job_info(server, job_name):
 
 		lcb_url = build_info['url']
 		lcb_result = build_info['result']
-		compose = [action['text'][13:-4] for action in build_actions if 'core_puddle' in action.get('text', '')]
+		compose = [action['html'][-42:-13] for action in build_actions if 'core_puddle' in action.get('html', '')]
```
Comment (amodi5) [inline functions.py:149]:
```suggestion
		compose = [str(action['html']).split('core_puddle:')[1].split('<')[0].strip() for action in build_actions if 'core_puddle' in action.get('html', '')]
```
Its not too pretty but saves going through the whole parsing routine with HTMLParser or Beautifulsoup, so this should work for now

Comment (amodi5) [reply functions.py:149]: Tested it on all supported versions and got the desired ouput, in previous patch the versions 13 and 10 were not showing proper results, Example: ck">core_puddle: 2020-08-05.1 for one of the OSP 13 jobs

Comment (nathan-weinberg) [reply functions.py:149]: Ack, will update the patch with this line

**Comment by nathan-weinberg** [comment]
@amodi5 @agarcia64 Thanks for moving fast on this guys; going to give others the rest of the day to test and will merge tomorrow :+1:

**Pull request merged**